src/hooks/cashu/usePendingTransaction.ts

```diff
@@ -0,0 +1,24 @@
+import { PendingMintQuote } from '@/types';
+import { MintQuoteResponse } from '@cashu/cashu-ts';
+import { useLocalStorage } from 'usehooks-ts';
+
+export const usePendingTransaction = () => {
+   const [pendingMintQuotes, setPendingMintQuotes] = useLocalStorage<Array<PendingMintQuote>>(
```

Comment:
nit so feel free to ignore but it's shorter to write `PendingMintQuote[]` than `Array<PendingMintQuote>`